src/main/java/net/openhft/chronicle/queue/RollCycles.java

```diff
@@ -264,6 +273,11 @@ public int lengthInMillis() {
         return this.lengthInMillis;
     }
 
+    @Override
+    public int defaultEpoch() {
```

Comment:
why add this?

Reply:
So a RollCycle could define an implied epoch, needed for WEEKLY rolling.